tests/unit/test_value.c

```diff
@@ -509,59 +510,6 @@ SENTRY_TEST(value_wrong_type)
     TEST_CHECK(sentry_value_get_length(val) == 0);
 }
 
-SENTRY_TEST(value_collections_leak)
```

Comment:
Removed this test since it no longer makes sense with the new ringbuffer append. Maybe this should be replaced by a similar test (e.g. one where we add objects to the ringbuffer & make sure they get refcount reduced when they get overwritten)

Reply:
Yes, that makes sense. I am unsure about a _new_ test as long as we test breadcrumb overflow _somewhere_ because Valgrind or ASAN would scream if the ring buffer append didn't correctly ref-count values.

Reply:
Added a new test `SENTRY_TEST(value_ringbuffer)` which aims to check exactly the behaviour I described above. If you agree on the test structure/contents I can mark this point as resolved ^